data/json/recipes/armor/suit.json

```diff
@@ -438,10 +438,10 @@
     "reversible": true,
     "autolearn": true,
     "book_learn": [ [ "recipe_melee", 2 ] ],
-    "using": [ [ "cordage", 6 ], [ "mixed_steel", 5 ] ],
-    "components": [ [ [ "cotton_patchwork", 28 ] ] ],
-    "proficiencies": [ { "proficiency": "prof_closures" }, { "proficiency": "prof_articulation" } ],
-    "qualities": [ { "id": "CUT", "level": 2 }, { "id": "HAMMER", "level": 1 } ]
+    "using": [ [ "cordage", 6 ], [ "mixed_steel", 3 ] ],
+    "components": [ [ [ "cotton_patchwork", 20 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_armorsmithing", "fail_multiplier": 1 }, { "proficiency": "prof_closures" }, { "proficiency": "prof_articulation" } ],
```

Comment:
Pretty sure this line is too long. Run it through a linter and see what you get. We physically can't merge it until the lint is passed.

Reply:
Thanks for the response. I tried linting it again and the latest changes are at https://github.com/CleverRaven/Cataclysm-DDA/pull/58872.

Reply:
I've closed this branch to avoid confusion. Further updates will be on the one above.